test/cypress/e2e/multi-datasources/multi_datasources_enabled.spec.js

```diff
@@ -0,0 +1,244 @@
+/*
+ *   Copyright OpenSearch Contributors
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+const externalTitle = '9202';
+const createDataSource = () => {
+  return cy.request({
+    method: 'POST',
+    url: `${Cypress.config('baseUrl')}/api/saved_objects/data-source`,
+    headers: {
+      'osd-xsrf': true,
+    },
+    body: {
+      attributes: {
+        title: externalTitle,
+        endpoint: 'https://localhost:9202',
+        auth: {
+          type: 'username_password',
+          credentials: {
+            username: 'admin',
+            password: 'myStrongPassword123!',
```

Comment:
Can we pick this up from a variable instead of hard coding a string?

Reply:
Yup - good suggestion, and I did the same.